### PR TITLE
fix: should without lock when persona sign

### PR DESCRIPTION
**Pull request by Lanttcat**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by wenluomask** [comment]
Refer to Jira: https://mask.atlassian.net/browse/MASK-737

**Comment by wenluomask** [comment]
If wallet is locked, I sign Persona first and then sign wallet next, it will be failed to sigh.
https://user-images.githubusercontent.com/85149605/156345310-b57e4c9a-978c-4118-a6c7-fedb80271adb.mov

**Comment by wenluomask** [comment]
Persona signature page should only show Persona tab;
Screen Shot 2022-03-03 at 1 56 18 PM

**Comment by wenluomask** [comment]
Checked.

**Review by UncleBill — DISMISSED** [review]

File: packages/mask/background/services/helper/popup-opener.ts [hunk 1]
```diff
@@ -23,13 +23,20 @@ export async function openPopupWindow(route?: PopupRoutes, params?: Record<strin
     if (popup) {
         await browser.windows.update(currentPopupWindowId, { focused: true })
     } else {
+        const exclusionDetectLocked = [PopupRoutes.PersonaSignRequest]
         const locked = await isLocked()
```
Comment (guanbinrui) [inline packages/mask/background/services/helper/popup-opener.ts:29]:
```ts
const shouldUnlockWallet = await isLocked() && ![PopupRoutes.PersonaSignRequest].includes(route ?? PopupRoutes.Wallet)
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**